modules/ROOT/pages/java-support.adoc

```diff
@@ -33,6 +33,25 @@ The maximum supported version for running your applications is Java 17 for Mule
 
 For more information about Mule release cadences, see xref:release-notes::mule-runtime/lts-edge-release-cadence.adoc[].
 
+== Steps for Upgrade 
+
+Follow these steps to upgrade to Java 17:
+
+. Learn about <<application-performance, application performance>> when upgrading to Java 17. 
+. Conduct an analysis of your current setup to pinpoint compatibility issues or deprecated elements.
```

Comment:
I don't understand the meaning or purpose of this sentense:

Analysis looking for what? 
How is a non java expert supposed to do that?
What is a deprecated element in this context? A Java feature used in custom code? A mule component?